base/applications/taskmgr/procpage.c

```diff
@@ -931,3 +936,300 @@ int CALLBACK ProcessPageCompareFunc(LPARAM lParam1, LPARAM lParam2, LPARAM lPara
     }
     return ret;
 }
+
+static BOOL DevicePathToDosPath(LPWSTR lpPath, DWORD dwLength)
+{
+    WCHAR *pszPath;
+    WCHAR cDrive;
+    BOOL bSuccess = FALSE;
+
+    /* Check if lpPath is a device path */
+    if (_wcsnicmp(lpPath, L"\\Device\\", 8) != 0)
+    {
+        return FALSE;
+    }
+
+    pszPath = HeapAlloc(GetProcessHeap(), 0, dwLength * sizeof(WCHAR));
```

Comment:
Seeing how you use `pszPath` and `lpPath`, you could directly skip that and in the `StringCchPrintfW` call below, use `lpPath`.

Reply:
OK.